scripts/build-catalogue.in

```diff
@@ -39,6 +39,16 @@ def parse_arguments():
                         type=str,
                         help='Catalogue name.')
 
+    parser.add_argument('--raw',
+                        metavar='raw',
+                        nargs='+',
+                        default=[],
+                        type=str,
+                        help='''Advanced: Raw mechanisms as C++ files. Per <name> the
+files <name>.hpp, <name>_cpu.cpp must be present
+in the target directory and with GPU support
+also <name>_gpu.cpp and <name>_gpu.cu''')
```

Comment:
- `_cpu.cpp` is no longer needed for GPU build. 
- Can we replace `raw` with `cxx`?

Reply:
1. Fixed
2. I'd rather keep the abrasive name. It's meant as a deterrent in a sense and the experience really is 'raw'.
    Users need to
    - observe our naming conventions for files, functions, and more.
    - manually namespace their kernels
    - juggle function pointers
    - manually set parameters counts and names